src/main/java/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package task;
+
+public class Task {
+    private static int numTask = 0;
+    private static final int numDone = 0;
+    private String taskName;
+    private boolean isDone;
+    private char type;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.isDone = false;
+        this.type = 'T';
+        numTask++;//increments the number of task
+    }
+
+    public Task() {
+        this.taskName = "";
+        this.isDone = false;
+    }
+
+    public static int getNumTask() {
+        return numTask;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public void setTaskName(String taskName) {
+        this.taskName = taskName;
+    }
+
+    public boolean getStatus() {
+        return isDone;
+    }
+
+    public void setDone() {
```

Comment:
To be more consistent, I recommend naming this setter as setStatus to correspond to your getter above which is getStatus, and in fact what kind of status? I would think setDoneStatus and getDoneStatus are more descriptive